submissions/Oleksii_Drohachov/OOP-exercise/index.js

```diff
@@ -0,0 +1,73 @@
+class Creature {
+  constructor(species, name, gender, saying, friends) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.saying = saying;
+    this.friends = friends;
+  }
+
+  output() {
+    const { name, species, gender, saying, friends } = this;
+
+    return `${name}; ${species}; ${gender}; ${saying}; ${friends.join(", ")}`;
+  }
+}
+
+class Mammal extends Creature {
```

Comment:
> `Mammal` is the only class that extends `Creature`. Why do we need `Mammal`?

Any good reason for this?

Also, obviously there is no good reason to prefer 4 for legs. Decisions should be explainable.